ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/VulnerabilityStateTabs.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { CSSProperties } from 'react';
+import { Tabs, Tab, TabTitleText } from '@patternfly/react-core';
+
+import { vulnerabilityStates } from 'types/cve.proto';
+import useFeatureFlags from 'hooks/useFeatureFlags';
+import useURLStringUnion from 'hooks/useURLStringUnion';
+
+export type VulnerabilityStateTabsProps = {
+    className?: string;
+    titleOverrides?: {
+        observed?: string;
+        deferred?: string;
+        falsePositive?: string;
+    };
+};
+
+function VulnerabilityStateTabs({ className, titleOverrides }: VulnerabilityStateTabsProps) {
+    const { isFeatureFlagEnabled } = useFeatureFlags();
+    const isUnifiedDeferralsEnabled = isFeatureFlagEnabled('ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL');
+    const [vulnerabilityStateKey, setVulnerabilityStateKey] = useURLStringUnion(
+        'vulnerabilityState',
+        vulnerabilityStates
+    );
```

Comment:
@dvail since `useVulnerabilityState` has a lot of the same logic as here, should we try to use it here too?

Reply:
We could, although I think once the deferral feature flag is removed I will end up removing the `useVulnerabilityState` hook as well since it will just be a wrapper for the hook shown here. The only difference switching in this case is that the latter hook can currently return `undefined`, but I don't think that impacts this component since in the `undefined` case the Tabs won't render at all.